apps/web/src/stores/evaluationResultsByDocumentContent.ts

```diff
@@ -0,0 +1,75 @@
+import { useCallback } from 'react'
+
+import { EvaluationResultWithMetadata } from '@latitude-data/core/repositories'
+import { useToast } from '@latitude-data/web-ui'
+import useSWR, { SWRConfiguration } from 'swr'
+
+type SerializedEvaluationResult = Omit<
+  EvaluationResultWithMetadata,
+  'createdAt' | 'updatedAt'
+> & {
+  createdAt: string
+  updatedAt: string
+}
+
+const EMPTY_ARRAY: [] = []
+export default function useEvaluationResultsByDocumentContent(
+  {
+    evaluationId,
+    documentUuid,
+    commitUuid,
+    projectId,
+  }: {
+    evaluationId: number
+    documentUuid: string
+    commitUuid: string
+    projectId: number
+  },
+  { fallbackData }: SWRConfiguration = {},
+) {
+  const { toast } = useToast()
+
+  const fetcher = useCallback(async () => {
+    const response = await fetch(
+      `/api/documents/${projectId}/${commitUuid}/${documentUuid}/evaluationResultsByDocumentContent/${evaluationId}`,
```

Comment:
Let's start today doing `API_ROUTES`. There are 3 cases. Please start this good practice 🙏